{"files":[{"patch":"@@ -222,1 +222,4 @@\n-     * the VM will exit with a non-zero exit code.\n+     * the VM will exit with a non-zero exit code. To avoid the VM aborting due to an uncaught exception, clients\n+     * could wrap all code in the target method handle in a try\/catch block that catches any {@link Throwable}, for\n+     * instance by using the {@link java.lang.invoke.MethodHandles#catchException(MethodHandle, Class, MethodHandle)}\n+     * method handle combinator, and handle exceptions as desired in the corresponding catch block.\n","filename":"src\/jdk.incubator.foreign\/share\/classes\/jdk\/incubator\/foreign\/CLinker.java","additions":4,"deletions":1,"binary":false,"changes":5,"status":"modified"}]}